cpp/src/parquet/encoding_test.cc

```diff
@@ -1090,7 +1090,7 @@ typedef ::testing::Types<FloatType, DoubleType> ByteStreamSplitTypes;
 TYPED_TEST_SUITE(TestByteStreamSplitEncoding, ByteStreamSplitTypes);
 
 TYPED_TEST(TestByteStreamSplitEncoding, BasicRoundTrip) {
-  ASSERT_NO_FATAL_FAILURE(this->Execute(1000, 1));
+  ASSERT_NO_FATAL_FAILURE(this->Execute(1337, 1));
```

Comment:
What's the impact of this number?

Reply:
I wanted to select a number which is not divisible by 4 or 8 to guarantee that the last partially full block is handled by the scalar decode loop.
1337 was just a capricious whim. I could have also used 1003.

Reply:
OK, just add a comment to explain the choice of number

Reply:
Or, rather, test with a range of different lengths (say, 16 of them), if it doesn't make the test too long.

Reply:
Done.